2012/08/02 13:55:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testyii/index.php?r=gii
---
2012/08/02 13:58:57 [error] [exception.CException] exception 'CException' with message 'Alias "application.User" is invalid. Make sure it points to an existing PHP file.' in E:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 E:\xampp\htdocs\yii\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.Use...', true)
#1 E:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(43): CrudCode->validateModel('model', Array)
#2 E:\xampp\htdocs\yii\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 E:\xampp\htdocs\yii\framework\base\CModel.php(158): CValidator->validate(Object(CrudCode), NULL)
#4 E:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(158): CModel->validate()
#5 E:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#6 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 E:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CController->run('')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/crud')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/testyii/index.php?r=gii/crud
HTTP_REFERER=http://localhost/testyii/index.php?r=gii/crud
---
2012/08/02 14:04:59 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.admin.model.Categoryu" is invalid. Make sure it points to an existing PHP file.' in E:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 E:\xampp\htdocs\yii\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 E:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(43): CrudCode->validateModel('model', Array)
#2 E:\xampp\htdocs\yii\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 E:\xampp\htdocs\yii\framework\base\CModel.php(158): CValidator->validate(Object(CrudCode), NULL)
#4 E:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(158): CModel->validate()
#5 E:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#6 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 E:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CController->run('')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/crud')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/testyii/index.php?r=gii/crud
HTTP_REFERER=http://localhost/testyii/index.php?r=gii/crud
---
2012/08/02 14:09:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(292): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/category/...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/testyii/index.php?r=admin/category/admin
HTTP_REFERER=http://localhost/testyii/index.php?r=admin/category/index
---
2012/08/02 14:09:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "category".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('category')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testyii/index.php?r=category
---
2012/08/02 14:10:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(292): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/category/...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/testyii/index.php?r=admin/category/admin
HTTP_REFERER=http://localhost/testyii/index.php?r=admin/category/view&id=1
---
2012/08/02 14:12:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/login".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/login')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testyii/index.php?r=admin/login
---
2012/08/02 14:12:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/user".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/user')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testyii/index.php?r=admin/user
---
2012/08/02 14:13:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/login".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/login')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testyii/index.php?r=admin/login
---
2012/08/02 14:13:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/user".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/user')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/testyii/index.php?r=admin/user
---
2012/08/02 15:21:39 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.admin.model.Login" is invalid. Make sure it points to an existing PHP file.' in E:\xampp\htdocs\yii\framework\YiiBase.php:318
Stack trace:
#0 E:\xampp\htdocs\yii\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 E:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(43): CrudCode->validateModel('model', Array)
#2 E:\xampp\htdocs\yii\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 E:\xampp\htdocs\yii\framework\base\CModel.php(158): CValidator->validate(Object(CrudCode), NULL)
#4 E:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(158): CModel->validate()
#5 E:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#6 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 E:\xampp\htdocs\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CController->run('')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/crud')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/testyii/index.php?r=gii/crud
HTTP_REFERER=http://localhost/testyii/index.php?r=gii/crud
---
2012/08/04 06:51:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\testyii\protected\modules\admin\controllers\SubcategoryController.php:160
Stack trace:
#0 E:\xampp\htdocs\testyii\protected\modules\admin\controllers\SubcategoryController.php(54): SubcategoryController->loadModel('2')
#1 [internal function]: SubcategoryController->actionView('2')
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(SubcategoryController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(SubcategoryController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(292): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CController->run('view')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('admin/subcatego...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 E:\xampp\htdocs\testyii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/testyii/index.php?r=admin/subcategory/view&id=2
---
